graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -430,23 +435,17 @@ class DgsSchemaProvider(
                 logger.info("No schema files found, but a schema was provided as an TypeDefinitionRegistry")
                 arrayOf()
             } else {
-                logger.error("No schema files found. Define schemas in src/main/resources/$basedir/**/*.graphqls")
+                logger.error("No schema files found. Define schema locations with property dgs.graphql.schema-locations")
```

Comment:
**[NIT]** Here we do not know if there were either no schema files found or the locations was set incorrectly. I think we should somewhat "combine" the 2 messages. Something like: 
```java
"No schema files found in ${schemaLocations}. Define schema locations with property dgs.graphql.schema-locations"
```
wdyt?

Reply:
This is updated now.